cms/templates/certificates.html

```diff
@@ -92,7 +92,7 @@ <h3 class="title-3">${_("Working with Certificates")}</h3>
             <p>${_("To view a sample certificate, choose a course mode and select {em_start}Preview Certificate{em_end}.").format(em_start='<strong>', em_end="</strong>")}</p>
             
             <h3 class="title-3">${_("Issuing Certificates to Learners")}</h3>
-            <p>${_("To begin issuing certificates, a course team member with the Admin role selects {em_start}Activate{em_end}. Course team members without the Admin role cannot edit or delete an activated certificate.").format(em_start='<strong>', em_end="</strong>")}</p>
+            <p>${_("After you have configured and previewed your certificate, ask your edX partner manager to activate your certificate. When your certificate has been activated, you can begin issuing certificates to learners. Course team members without the Admin role cannot edit or delete an activated certificate.").format(em_start='<strong>', em_end="</strong>")}</p>
             <p>${_("{em_start}Do not{em_end} delete certificates after a course has started; learners who have already earned certificates will no longer be able to access them.").format(em_start="<strong>", em_end="</strong>")}</p>
```

Comment:
I think you can remove the format statement from the end of the line, since it's no longer being used.